pkg/conversion/convert.go

```diff
@@ -339,7 +342,27 @@ func convertWindupWhenToAnalyzer(windupWhen windup.When, where map[string]string
 	if windupWhen.Technologytagexists != nil {
 		// conditions = append(conditions, map[string]interface{}{"technology-tag-exists": nil})
 	}
-	return conditions
+	return conditions, customVars
+}
+
+func convertWhereToCustomVars(whereMap map[string]string, fullPattern string) []map[string]interface{} {
+	l := []map[string]interface{}{}
+	newString := fullPattern
+	for k, v := range whereMap {
+		newString = strings.ReplaceAll(newString, "{"+k+"}.", fmt.Sprintf("(?P<%s>%s.)?", k, v))
+		newString = strings.ReplaceAll(newString, "{"+k+"}", fmt.Sprintf("(?P<%s>%s)?", k, v))
```

Comment:
This won't ever match that pattern though right, since we already replaced it in the line above?

Reply:
no, because the `.` at the end there does some extra magic :)